app/src/main/java/org/schabi/newpipe/fragments/list/playlist/PlaylistFragment.java

```diff
@@ -284,8 +285,15 @@ public void handleResult(@NonNull final PlaylistInfo result) {
 
         imageLoader.displayImage(result.getUploaderAvatarUrl(), headerUploaderAvatar,
                 ImageDisplayConstants.DISPLAY_AVATAR_OPTIONS);
-        headerStreamCount.setText(getResources().getQuantityString(R.plurals.videos,
-                (int) result.getStreamCount(), (int) result.getStreamCount()));
+
+        int streamCount = (int) result.getStreamCount();
+        if (streamCount == PlaylistExtractor.MORE_THAN_100_ITEMS) {
```

Comment:
I would prefer a switch case here

Reply:
Mmmh you are right

Reply:
Done, also moved everything to a global `localizeStreamCount` function that is also used in PlaylistInfoItem and ChannelInfoItem and correctly localizes the item count.